src/components/plotly-modal/index.js

```diff
@@ -19,39 +19,46 @@ const PlotlyModal = ({ metadata, onToggle, visible }) => {
     return null;
   }
   return (
-    <div className="pipeline-plotly-modal">
-      <div className="pipeline-plot-modal__top">
-        <button
-          className="pipeline-plot-modal__back"
-          onClick={onCollapsePlotClick}
-        >
-          <BackIcon className="pipeline-plot-modal__back-icon"></BackIcon>
-          <span className="pipeline-plot-modal__back-text">Back</span>
-        </button>
-        <div className="pipeline-plot-modal__header">
-          <NodeIcon className="pipeline-plot-modal__icon" icon={nodeTypeIcon} />
-          <span className="pipeline-plot-modal__title">
-            {metadata.node.name}
-          </span>
+    <>
+      {metadata && (
```

Comment:
This check works most of the time but there seems to be a case where it doesn't:

1. Open viz and click on the "Uplift Report Figures" node
2. Expand the Plotly graph to full-size
3. Hit the escape key
4. Click any other node that doesn't have a Plotly chart in the metadata panel

You should see an error. 

I think if you're just a little more specific we should be set:

```suggestion
      {metadata && metadata.plot && (
```